doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,218 @@
+==============================================================================
+Flow around a Finite Cylindrical Particle Using the Sharp Interface Method
+==============================================================================
+
+This example simulates the flow around a perpendicular (:math:`\theta = 90^\circ`) cylinder at low Reynolds number (:math:`Re \in [0.01, 5]`) and for an aspect ratio (defined as :math:`Ar = L/D`) of 2.
+
+----------------------------------
+Features
+----------------------------------
+
+- Solver: ``lethe-fluid-sharp`` (with Q1-Q1) 
+- Steady-state problem
+- Static non-spherical particle
+- Force calculation validation
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+- Parameter file: ``examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.prm``
+
+------------------------
+Description of the Case
+------------------------
+
+This example showcases the use of sharp immersed boundary conditions to simulate flow around non-spherical objects, in this case a cylindrical particle perpendicular to the flow. 
+
+This example focuses on cylindrical particles of small aspect ratio :math:`Ar` (:math:`Ar = L/D = 2`) at low Reynolds numbers ranging from 0.01 to 5. To validate the simulation results, the drag coefficient is 
+compared to the expected results from the literature. 
+
+.. image:: images/flow-around-particle-sharp-case.png
+    :alt: Simulation schematic
+    :align: center
+
+---------------
+Parameter File
+---------------
+
+Mesh
+~~~~
+
+As shown in the previous figure, the flow around the partice is modelled using a prismatic channel which size depends on the size of the particle. In this case, we consider a particle with a diameter of :math:`D = 1`, therefore we have :math:`L = 2`. 
+The domain, of size :math:`32 L \times 16 L \times 16 L`, was found to produce results independant of domain sizes with a low computational cost. 
+
+The mesh is first defined coarsely away from the particle with elements of size :math:`L \times L \times L`. The particle is positionned at :math:`3/8` of the domain length avoid boundary interactions and allow the flow to develop both behind and in front of the particle.
```

Comment:
```suggestion
The mesh is first defined coarsely away from the particle with elements of size :math:`L \times L \times L`. The particle is positionned at :math:`3/8` of the domain length to avoid boundary interactions and allow the flow to develop both behind and in front of the particle.
```